docs/dyndns/configure.rst

```diff
@@ -108,16 +109,21 @@ To install ddclient, run ``sudo apt-get install ddclient``. If a configuration
 dialog does not appear automatically, use ``sudo dpkg-reconfigure ddclient`` to
 start the configuration process.
 
-In the configuration process, select "other" dynamic DNS service provider, and
-enter ``update.dedyn.io`` as the dynamic DNS server. Next, tell ddclient to use
-the "dyndns2" protocol to perform updates. Afterwards, enter the username and
-the token secret that you received during registration. Last, tell ddclient how to
-detect your IP address, your domain name and the update interval.
+In the configuration process, select "other" dynamic DNS service provider. 
+Next, tell ddclient to use the "dyndns2" protocol to perform updates.
+Enter ``update.dedyn.io`` as the dynamic DNS service. Afterwards, enter the domain 
+you want to update as the username.
+As your password, use a token that you generated under ``https://desec.io/tokens``.
```

Comment:
For folks signing up with a .dedyn.io domain (currently suspended), a token is displayed after signing up. Telling them to create a token would confuse them.

Reply:
hmm... so maybe something more generic?
Something like "use your access token as password"

Reply:
Perhaps, but that's not significantly different from the current wording.